src/docs/output-targets/custom-elements.md

```diff
@@ -41,18 +41,17 @@ customElements.define('hello-world', HelloWorld);
 
 ### autoDefineCustomElements
 
-By default, consumers of the `custom-elements` output target need to either register each Stencil component in the 
-bundle manually, or call a convenience method, `defineCustomElements()`, that is exported as a part of the bundle to
-define every component in the bundle. This behavior can be cumbersome, especially when only a handful of components are 
-needed and/or those components have several child components (and their children have children, etc.).
-
-Setting this flag to `true` will recursively define all children components for a Stencil component when it is
-registered. Users of this flag should note that this may increase their bundle size by automatically defining &
-registering child components.
+By default, consumers of the `dist-custom-elements` output target need to
+register each Stencil component that they want to use in the bundle manually.
+Setting this flag to `true` will add a call to the `defineCustomElement` helper
```

Comment:
This sentence reads a little off to me. I think that

> helper function for each function to it's module, which will be called when it is exported.

is a little difficult to follow.

I wonder if we could rephrase this such that we can achieve the same affect with:
```suggestion
Setting this flag to `true` will define a component automatically with the
`CustomElementRegistry`. It will also recursively define all child components
of a component.
```

We actually don't ever mention `defineCustomElement` anywhere else on the page 😬 We probably should, but that's another issue for another day.